packages/protocol/contracts/layer1/core/libs/LibManifest.sol

```diff
@@ -33,6 +33,16 @@ library LibManifest {
     /// processed.
     uint256 internal constant BOND_PROCESSING_DELAY = 6;
 
+    /// @notice The initial base fee for the first Shasta block.
+    uint256 internal constant INITIAL_BASE_FEE = 0.025 gwei;
+
+    /// @notice The minimum base fee (inclusive) after Shasta fork
+    uint256 internal constant MIN_BASE_FEE = 0.005 gwei;
+
+    /// @notice The maximum base fee (inclusive) after Shasta fork
+    uint256 internal constant MAX_BASE_FEE = 10 gwei;
```

Comment:
@davidtaikocha do you need these values?

Reply:
Inside this contract? No, we can move them into `Derivation.md`.